### DISCUSSION TITLE
Example 13 PySCF/Verlet Alg

### DISCUSSION THREAD

**Discussion by HenryMakowitz** [Quantum-Dynamics-Hub/libra-code#131] [Q&A] [answered]
Your Example 13 ; PySCF code with  Verlet Algorithm ;   I am assuming this is for AIMD ? Where can I find more documentation ?? 
I'm interested in AIMD DFT and AIMD TD-DFT with the PySCF Code.

**Comment by alexvakimov** [comment, marked as answer]
Dear Henry, 

Yes, that's correct - it is for AIMD. What kind of documentation are you looking for? If you want to learn more about the PySCF code, I'd recommend their [website](https://pyscf.org/) as well as the documentation that is supplied with the code.  As for the Verlet algorithm - my hope was that it would be rather self-explanatory. The code uses the Libra package - you can find more detailed and revised examples [here](https://github.com/compchem-cybertraining/Tutorials_Libra). Please let me know if you have any further questions. 

Best,
Alexey
